### PR TITLE
test: Run all tests in a separate ClickHouse database

### PR DESCRIPTION
**Pull request by lynnagara**
We should run our tests in a separate ClickHouse database. This will
allow us to keep tests better separated from dev and simplify the
table name logic and not rely on preprepending the table names with `test_`.

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
     setup_sentry()
+
+    # There is only one cluster in test, so fetch the host from there.
+    cluster = settings.CLUSTERS[0]
+
+    connection = ClickhousePool(
+        cluster["host"], cluster["port"], "default", "", "default",
+    )
+
+    connection.execute("CREATE DATABASE IF NOT EXISTS snuba_test;")
```
Comment (fpacifici) [inline tests/conftest.py:20]: shouldn't you pick the name from the cluster i the settings ? I don't think there is the need to hardcode it in two places.

Also, as we drop the tables at the end we should probably drop the testingn DB as well.

**Review by fpacifici — CHANGES_REQUESTED** [review]
Please see my comment on conftest.py otherwise, this looks good, thanks

File: tests/conftest.py (hunk 1 above)
Comment (lynnagara) [reply tests/conftest.py]: Updated the database name. We don't currently drop the tables at the end. Would like to tackle that part separately.

Comment (fpacifici) [reply tests/conftest.py]: ok sure

**Review by fpacifici — APPROVED** [review]

**Pull request merged**